packages/automerge-repo/src/helpers/pause.ts

```diff
@@ -0,0 +1,2 @@
+export const pause = (t = 100) =>
```

Comment:
we don't use this with an argument anywhere -- we should make the default 0 or just not support having an argument.

even moreso (and maybe not in this PR) we should probably just... not do this at all.

Reply:
✅